sql/src/main/java/io/crate/metadata/sys/SysNodesTableInfo.java

```diff
@@ -241,10 +241,10 @@
             .put(SysNodesTableInfo.Columns.HEAP, NodeHeapStatsExpression::new)
             .put(SysNodesTableInfo.Columns.VERSION, NodeVersionStatsExpression::new)
             .put(SysNodesTableInfo.Columns.THREAD_POOLS, NodeThreadPoolsExpression::new)
-            .put(SysNodesTableInfo.Columns.THREAD_POOLS_NAME, () -> new NodeStatsThreadPoolExpression<String>() {
+            .put(SysNodesTableInfo.Columns.THREAD_POOLS_NAME, () -> new NodeStatsThreadPoolExpression<BytesRef>() {
                 @Override
-                protected String valueForItem(Map.Entry<String, ThreadPools.ThreadPoolExecutorContext> input) {
-                    return input.getKey();
+                protected BytesRef valueForItem(Map.Entry<String, ThreadPools.ThreadPoolExecutorContext> input) {
```

Comment:
wouldn't it be better to just change the ThreadPools implementation to use BytesRef keys instead of Strings?

Reply:
that's also an option. would make sense not only for threadpool expression, but for all array type expression where we deal with maps

Reply:
@seut We can do but this should be a separate commit that will only stay in master since it will change the streaming between nodes.

Reply:
@matriv yep right, thx.